robosat/tools/train.py

```diff
@@ -46,11 +46,23 @@ def add_parser(subparser):
     parser.add_argument("--model", type=str, required=True, help="path to model configuration file")
     parser.add_argument("--dataset", type=str, required=True, help="path to dataset configuration file")
     parser.add_argument("--checkpoint", type=str, required=False, help="path to a model checkpoint (to retrain)")
+    parser.add_argument("--resume", type=bool, default=False, help="resume training or fine-tuning (if checkpoint)")
     parser.add_argument("--workers", type=int, default=1, help="number of workers pre-processing images")
 
     parser.set_defaults(func=main)
 
 
+class Log:
+    def __init__(self, path):
+        self.fp = open(path, "a")
+
+    def log(self, msg):
+        print(msg)
+        assert self.fp, "Unable to write in log file"
+        self.fp.write(msg + os.linesep)
+        self.fp.flush()
```

Comment:
Probably a good idea to create a new Python module for everything logging related.

If we make it a bit more generic we could also change the impl. e.g. to a `csv.writer` or later on think about hooking up e.g. tensorboard.

Reply:
Could be yes,
I just pushed a new separated Log module.
At this point, it still quite close the initial implementation.